docs/sphinx/quickstart.rst

```diff
@@ -10,206 +10,41 @@
-
-.. code-block:: sh
-
-    make tests
-
-To run the tests:
-
-.. code-block:: sh
-
-    make test
-
-To control which tests to run use ``ctest``:
-
-* To run single tests, for example a test for ``for_loop``:
-
-.. code-block:: sh
-
-    ctest --output-on-failure -R tests.unit.modules.algorithms.for_loop
-
-* To run a whole group of tests:
-
-.. code-block:: sh
-
-    ctest --output-on-failure -R tests.unit
-
-Run examples
-------------
-
-If you did not run ``make install`` earlier, do so now or build the
-``hello_world_1`` example by running:
-
-.. code-block:: sh
-
-   make hello_world_1
-
-|hpx| executables end up in the ``bin`` directory in your build directory. You
-can now run ``hello_world_1`` and should see the following output:
-
-.. code-block:: sh
-
-   ./bin/hello_world_1
-   Hello World!
-
-You've just run an example which prints ``Hello World!`` from the |hpx| runtime.
-The source for the example is in ``examples/quickstart/hello_world_1.cpp``. The
-``hello_world_distributed`` example (also available in the
-``examples/quickstart`` directory) is a distributed hello world program, which is
-described in :ref:`examples_hello_world`. It provides a gentle introduction to
-the distributed aspects of |hpx|.
-
-.. tip::
-
-   Most build targets in |hpx| have two names: a simple name and
-   a hierarchical name corresponding to what type of example or
-   test the target is. If you are developing |hpx| it is often helpful to run
-   ``make help`` to get a list of available targets. For example, ``make help |
-   grep hello_world`` outputs the following:
+   Another way to install |hpx| is using `Spack <https://spack.readthedocs.io/en/latest/>`_:
 
    .. code-block:: sh
 
-      ... examples.quickstart.hello_world_2
-      ... hello_world_2
-      ... examples.quickstart.hello_world_1
-      ... hello_world_1
-      ... examples.quickstart.hello_world_distributed
-      ... hello_world_distributed
+      spack install hpx
 
-   It is also possible to build, for instance, all quickstart examples using ``make
-   examples.quickstart``.
+#. **Fedora**
 
-Installing and building |hpx| via vcpkg
-=======================================
+   Installation can be done with `Fedora <https://fedoraproject.org/wiki/DNF>`_ as well:
 
-You can download and install |hpx| using the `vcpkg <https://github.com/Microsoft/vcpkg>` 
-dependency manager:
+   .. code-block:: sh
 
-.. code-block:: sh
+      dnf install hpx*
 
-    git clone https://github.com/Microsoft/vcpkg.git
-    cd vcpkg
-    ./bootstrap-vcpkg.sh
-    ./vcpkg integrate install
-    vcpkg install hpx
+#. **Arch Linux**
 
-The |hpx| port in vcpkg is kept up to date by Microsoft team members and community 
-contributors. If the version is out of date, please `create an issue or pull request 
-<https://github.com/Microsoft/vcpkg>` on the vcpkg repository.
+   |hpx| is available in the `Arch User Repository (AUR) <https://wiki.archlinux.org/title/Arch_User_Repository>`_ as ``hpx`` too.
```

Comment:
I would still include a link to the build instructions in the manual as the last thing in case a user can't or don't want to use the above for some reason.

Reply:
Yes, you are absolutely right.

Reply:
I seem to remember this being added earlier as well. Probably also gone for some reason? :/